usaspending_api/api_contracts/search_filters.md

```diff
@@ -113,9 +113,15 @@ Keys in a location object include:
     * If `county` is provided, a `district` value *must never* be provided.
 * **city** - string city name
     * If no `state` is provided, this will return results for all cities in any state with the provided name
-* **district** - a 2 character code indicating the congressional district
-    * If `district` is provided, a `state` must always be provided as well.
-    * If `district` is provided, a `county` *must never* be provided.
+* **district** - Deprecated. Use `district_original`.
+* **district_original** - a 2 character code indicating the congressional district
+    * When provided, a `state` must always be provided as well.
+    * When provided, a `county` *must never* be provided.
+    * When provided, `country` must always be "USA".
+* **district_current** - a 2 character code indicating the current congressional district
+    * When provided, a `state` must always be provided as well.
+    * When provided, a `county` *must never* be provided.
+    * When provided, `country` must always be "USA".
```

Comment:
Might want to make a note that when a `district_current` is provided, a `district_original` must not be provided, and vice versa

Reply:
Good point. Will do.

Reply:
Fixed